### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by serbernar**
- added .idea directory to gitignore
- changed quotes in docstrings
- changed format ot f-strings
- Added to license electumx authors and remove from top of python files

### CODE REVIEW

**Comment by kyuupichan** [comment]
No, the copyright notices in lib/ stay as they are.  That's because those files are useful on their own and not as part of the whole.  I added explicit copyright notices to the files, so people could copy/paste if they wanted.

Changing ''' to """ doesn't seem to achieve anything other than churn.

The rest might be interesting, but I'd need to see it with the above unchanged.

**Comment by serbernar** [comment]
Okay, i'll fix it in the next commit, 10 min, pls

**Comment by serbernar** [comment]
I got error on travis

File: lib/enum.py [hunk 1]
```diff
@@ -4,38 +4,37 @@
...
-        for x in enumList:
+        for x in enum_list:
             if isinstance(x, tuple):
                 x, i = x
             if not isinstance(x, str):
-                raise EnumError("enum name {} not a string".format(x))
+                raise EnumError(f"enum name {x} not a string")
             if not isinstance(i, int):
-                raise EnumError("enum value {} not an integer".format(i))
+                raise EnumError(f"enum value {x} not an integer")
```
Comment (bauerj) [inline lib/enum.py]: That doesn't look right.

File: lib/peer.py [hunk 2]
```diff
@@ -43,9 +23,9 @@ class Peer(object):
     DEFAULT_PORTS = {}
 
     def __init__(self, host, features, source='unknown', ip_addr=None,
-                 good_ports=[], last_good=0, last_try=0, try_count=0):
-        '''Create a peer given a host name (or IP address as a string),
-        a dictionary of features, and a record of the source.'''
+                 good_ports=list(), last_good=0, last_try=0, try_count=0):
```
Comment (bauerj) [inline lib/peer.py]: Why is this necessary? The default value is never changed anyway.

**Comment by bauerj** [comment]
I also don't think that changing from triple single quotes to triple double quotes is worth the huge changeset. It only makes your PR harder to review.

**Comment by serbernar** [comment]
But we have PEP, and [PEP257](https://www.python.org/dev/peps/pep-0257/) said. It's do hard for reviewers, but project will not returned to this problem, imho.

**Comment by serbernar** [comment]
I don't understand: why travis doesn't work.
I got
```
1.62s$ sudo add-apt-repository -y ppa:giskou/librocksdb
gpg: keyring `/tmp/tmpl66d61d1/secring.gpg' created
gpg: keyring `/tmp/tmpl66d61d1/pubring.gpg' created
gpg: requesting key E7E01756 from hkp server keyserver.ubuntu.com
gpg: /tmp/tmpl66d61d1/trustdb.gpg: trustdb created
gpg: key E7E01756: public key "Launchpad PPA for Giannis Skoulis" imported
gpg: Total number processed: 1
gpg:               imported: 1  (RSA: 1)
OK
4.95s$ sudo apt-get -qq update
W: http://dl.hhvm.com/ubuntu/dists/trusty/InRelease: Signature by key 36AEF64D0207E7EEE352D4875A16E7281BE7A449 uses weak digest algorithm (SHA1)
W: http://ppa.launchpad.net/giskou/librocksdb/ubuntu/dists/trusty/InRelease: Signature by key A82B7A83D88C9A39DDEF81B729226849E7E01756 uses weak digest algorithm (SHA1)
W: The repository 'https://packagecloud.io/basho/riak/ubuntu trusty Release' does not have a Release file.
W: http://ppa.launchpad.net/couchdb/stable/ubuntu/dists/trusty/Release.gpg: Signature by key 15866BAFD9BCC4F3C1E0DFC7D69548E1C17EAB57 uses weak digest algorithm (SHA1)
E: Failed to fetch https://packagecloud.io/basho/riak/ubuntu/dists/trusty/main/source/Sources  401  Unauthorized
E: Failed to fetch https://packagecloud.io/basho/riak/ubuntu/dists/trusty/main/binary-amd64/Packages  401  Unauthorized
E: Failed to fetch https://packagecloud.io/basho/riak/ubuntu/dists/trusty/main/binary-i386/Packages  401  Unauthorized
E: Some index files failed to download. They have been ignored, or old ones used instead.
```

**Comment by bauerj** [comment]
Sure but "the PEP contains conventions, not laws or syntax".

> "A universal convention supplies all of maintainability, clarity, consistency, and a foundation for good programming habits too. What it doesn't do is insist that you follow it against your will. That's Python!"

—Tim Peters on comp.lang.python, 2001-06-16

**Comment by bauerj** [comment]
See: https://www.traviscistatus.com/incidents/77q4ts757mzs

**Comment by serbernar** [comment]
Okay, I am understood my mistakes and i'll send new PR tomorrow again. Sorry.

**Comment by kyuupichan** [comment]
Please keep the changes small and/or of one kind. It makes it easier to review.
